Signal/src/ViewControllers/DebugUI/DebugUIMessages.m

```diff
@@ -994,8 +1038,8 @@ + (void)sendFakeMessages:(NSUInteger)counter
                     [[TSOutgoingMessage alloc] initWithTimestamp:[NSDate ows_millisecondTimeStamp]
                                                         inThread:thread
                                                      messageBody:randomText];
-                DDLogError(@"%@ sendFakeMessages outgoing timestamp: %llu.", self.logTag, message.timestamp);
                 [message saveWithTransaction:transaction];
+                [message updateWithMessageState:TSOutgoingMessageStateUnsent transaction:transaction];
```

Comment:
Not super concerned, but curious: why mark them as unsent?

Reply:
Because otherwise `OWSFailedMessagesJob` bogs down the app upon next launch.  The point of these tools isn't to stress test `OWSFailedMessagesJob`.

Reply:
ah, that makes sense. I would not have guess that this was a perf optimization. Might be worth a code comment.